packages/terra-consumer-layout/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ChangeLog
 
+# 0.1.7 - (October 26, 2017)
+
+### Changed
+- Updating layout component to include terra-alert component
+- Updating DexLayout to include site alert example
+- terra-alert component used for site alert
```

Comment:
Can you clarify what changes are to prod, and what for tests.  ( Prod, only layout changed)